test/GithubApi.Put.test.js

```diff
@@ -0,0 +1,35 @@
+// Libraries
+const { expect } = require('chai');
+const chaiSubset = require('chai-subset');
+const chai = require('chai');
+// eslint-disable-next-line no-undef
```

Comment:
Try to avoid to disable eslint rules. Instad, check why they are showing and refactor what is needed to avoid them :)